.circleci/config.yml

```diff
@@ -168,8 +170,304 @@ jobs:
           command: |
             echo "Dummy job."
 
+  go-lint-test:
+    parameters:
+      working_directory:
+        description: Working directory
+        type: string
+    docker:
+      - image: mslipper/go-builder:latest
```

Comment:
We should get these into the optimism namespace at some point in the near future

Reply:
I can do this now.